src/app/app-routing/app-routing.module.ts

```diff
@@ -1,9 +1,15 @@
 import {NgModule} from '@angular/core';
 import {RouterModule, Routes} from '@angular/router';
 import {HistoryPageComponent} from '../history/history-page/history-page.component';
+import { MainPageLightComponent } from '../main/main-page-light/main-page-light.component';
+import { MainPageScoreboardComponent } from '../main/main-page-scoreboard/main-page-scoreboard.component';
+import {UserSettings} from '../user-settings';
 import {CenterViewComponent} from '../main/center-view/center-view.component';
 
+
+const isDarkMode = UserSettings.getTheme() == 'scoreboard' ;
```

Comment:
We can make a real dark theme for the app, so I think better name variables as isScoreboard